simulator/scheduler/config/plugin.go

```diff
@@ -29,275 +42,21 @@ func InTreeMultiPointPluginSet() (configv1.PluginSet, error) {
-// in-tree plugins and your original plugins listed in OutOfTreeFilterPlugins.
-func RegisteredFilterPlugins() ([]configv1.Plugin, error) {
-	def, err := InTreeFilterPluginSet()
-	if err != nil {
-		return nil, xerrors.Errorf("get default filter plugins: %w", err)
-	}
-
-	return append(def.Enabled, OutOfTreeFilterPlugins()...), nil
-}
-
-func InTreeFilterPluginSet() (configv1.PluginSet, error) {
-	defaultConfig, err := DefaultSchedulerConfig()
-	if err != nil || len(defaultConfig.Profiles) != 1 {
-		// default Config should only have default-scheduler configuration.
-		return configv1.PluginSet{}, xerrors.Errorf("get default scheduler configuration: %w", err)
-	}
-	return defaultConfig.Profiles[0].Plugins.Filter, nil
-}
-
-func OutOfTreeFilterPlugins() []configv1.Plugin {
-	return []configv1.Plugin{
-		// Note: add your filter plugins here.
-	}
-}
-
-func RegisteredPostFilterPlugins() ([]configv1.Plugin, error) {
-	def, err := InTreePostFilterPluginSet()
-	if err != nil {
-		return nil, xerrors.Errorf("get default post filter plugins: %w", err)
-	}
-	return append(def.Enabled, OutOfTreePostFilterPlugins()...), nil
-}
-
-func InTreePostFilterPluginSet() (configv1.PluginSet, error) {
-	defaultConfig, err := DefaultSchedulerConfig()
-	if err != nil || len(defaultConfig.Profiles) != 1 {
-		// default Config should only have default-scheduler configuration.
-		return configv1.PluginSet{}, xerrors.Errorf("get default scheduler configuration: %w", err)
-	}
-	return defaultConfig.Profiles[0].Plugins.PostFilter, nil
-}
-
-func OutOfTreePostFilterPlugins() []configv1.Plugin {
-	return []configv1.Plugin{
-		// Note: add your post filter plugins here.
-	}
-}
-
-// RegisteredScorePlugins returns all registered plugins.
-// in-tree plugins and your original plugins listed in OutOfTreeScorePlugins.
-func RegisteredScorePlugins() ([]configv1.Plugin, error) {
-	def, err := InTreeScorePluginSet()
-	if err != nil {
-		return nil, xerrors.Errorf("get default score plugins: %w", err)
-	}
-
-	return append(def.Enabled, OutOfTreeScorePlugins()...), nil
-}
-
-func InTreeScorePluginSet() (configv1.PluginSet, error) {
-	defaultConfig, err := DefaultSchedulerConfig()
-	if err != nil || len(defaultConfig.Profiles) != 1 {
-		// default Config should only have default-scheduler configuration.
-		return configv1.PluginSet{}, xerrors.Errorf("get default scheduler configuration: %w", err)
-	}
-	return defaultConfig.Profiles[0].Plugins.Score, nil
-}
-
-func OutOfTreeScorePlugins() []configv1.Plugin {
-	return []configv1.Plugin{
-		// Note: add your score plugins here.
-	}
+func OutOfTreeMultiPointPluginNames() []string {
+	return registeredOutOfTreeMultiPointName
 }
 
 func InTreeRegistries() runtime.Registry {
 	return plugins.NewInTreeRegistry()
 }
 
 func OutOfTreeRegistries() runtime.Registry {
-	return runtime.Registry{
-		// Note: add your plugins registries here.
+	return outOfTreeRegistries
+}
+
+func SetOutOfTreeRegistries(r runtime.Registry) {
```

Comment:
It'll be used by the externalplugin to register out of tree plugins (= registries).